lib/ansible/cli/pull.py

```diff
@@ -234,7 +235,7 @@ def run(self):
         cmd = '%s/ansible %s %s -m %s -a "%s" all -l "%s"' % (bin_path, inv_opts, base_opts, self.options.module_name, repo_opts, limit_opts)
 
         for ev in self.options.extra_vars:
-            cmd += ' -e "%s"' % ev
+            cmd += ' -e "%s"' % shlex_quote(ev)
```

Comment:
I don't think this works correctly.  I think you also need to remove the double quotes inside of the format string.